test/distributed/test_functional_api.py

```diff
@@ -588,42 +571,17 @@ def allreduce(t, pg):
             world_size=8,
             store=FakeStore(),
         )
-        allreduce(torch.randn(8, device=self.device), pg=dist.group.WORLD)
-
-    @unittest.skipIf(not HAS_GPU, "Inductor+gpu needs triton and recent GPU arch")
-    @requires_nccl()
-    @with_comms()
-    def test_tracing_with_dce_code(self):
```

Comment:
I don't think you should delete this test.

Reply:
You are right, this deletion was not needed.

I have restored this test in : 7afa48037bfcdc0f2f8837cebcaefcfbf633de89

Thanks